create_coda_files/create_coda_files.py

```diff
@@ -58,32 +64,46 @@ def _open_scheme(filepath):
     traced_json_output_path = args.traced_json_output_path
     coda_output_path = args.coda_output_path
     coding_scheme_path = args.coding_scheme_path
-    if args.auto_cleaner not in AUTO_CLEANERS.keys():
-        auto_cleaner = args.auto_cleaner
-        auto_cleaner = AUTO_CLEANERS[auto_cleaner]
-    else:
+    if args.has_yes_no == 'True':
+        has_yes_no = True
+    elif args.has_yes_no == 'False':
+        has_yes_no = False
```

Comment:
This isn't a safe input parser - if we change the above, this will result in has_yes_no silently becoming None, which will then cause confusing behaviours when we if test it below.

Consider something like:

lookup_map = {
  "True" : True,
  "False" : False
}

has_yes_no  = lookup_map[args.has_yes_no]\

(I'm not entirely sure why we're doing this though - I assume you've discussed with Alexander)